account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -1007,26 +1009,38 @@ def _get_reconcile_line(
         )
         rates = []
         for vals in new_vals:
+            rate = False
             if vals["partner_id"] is False:
                 vals["partner_id"] = (False, self.partner_name)
-            reconcile_auxiliary_id, rate = self._compute_exchange_rate(
-                vals, line, reconcile_auxiliary_id
-            )
+            if vals.get("kind") not in ("suspense", "liquidity"):
+                reconcile_auxiliary_id, rate = self._compute_exchange_rate(
+                    vals, line, reconcile_auxiliary_id
+                )
```

Comment:
In case of payment with a foreign currency, the real rate, applied by the bank can't be the Odoo one.
So, Odoo tries to create a exchange line even before reconciliation. I don't think it make sense, so we avoid this logic for liquidity/suspense line.

I believe that before starting reconciliation, we always should have 1 liquidity line and one suspense line, not a third line which would make a mess during reconciliation.